IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/QuantitySets/Qto_SignalBaseQuantities/DocQuantitySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_SignalBaseQuantities" UniqueId="b02c9d87-835c-43bb-bedb-7b4a65013060">
+	<Quantities>
+		<DocQuantity xsi:nil="true" href="Weight_2ueoe8QyrBduWuJHsi21zb" />
```

Comment:
Does it make sense to add height and width here?

Reply:
difficult one its similar to my impact device comment the geometric meaning of the height and width is very diffenent depending on the particular modelled signal